2019-04-16 22:10:09,336 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 166011ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-16 22:37:54,911 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1625567ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-16 22:53:02,407 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 867483ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-16 23:09:41,318 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 978865ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-16 23:28:57,865 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1116535ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-16 23:45:07,464 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 929581ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 00:08:58,286 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1410811ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 00:21:39,911 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 721603ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 00:48:21,765 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1541783ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 00:54:39,385 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 357597ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 01:24:24,015 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1744619ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 01:26:24,651 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 80622ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 02:03:27,287 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 2202615ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 02:12:40,928 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 513618ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 02:37:16,538 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1435584ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 02:50:44,170 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 787619ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 03:10:03,786 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1119602ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 03:27:04,415 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 980611ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 03:47:55,045 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1230601ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 03:56:22,664 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 467600ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 04:22:38,004 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1535318ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 04:31:32,630 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 514609ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 05:01:26,244 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1753597ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 05:06:35,875 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 269613ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 05:37:56,494 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1860614ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 05:53:22,114 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 885616ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 06:13:46,732 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1184599ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 06:22:44,357 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 517615ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 06:48:02,835 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1458458ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 06:57:44,401 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 561561ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 07:23:20,025 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1495613ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 07:29:27,642 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 327610ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 07:55:06,283 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1518629ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 08:02:44,927 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 418594ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 08:32:28,559 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1743613ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 08:46:16,282 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 807719ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 09:04:57,908 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1081617ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 09:17:19,529 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 701609ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 09:40:30,282 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 1370748ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

2019-04-17 09:42:12,523 ERROR 81333 nodejs.ClusterClientNoResponseError: client no response in 62224ms exceeding maxIdleTime 60000ms, maybe the connection is close on other side.
    at Timeout.Leader._heartbeatTimer.setInterval [as _onTimeout] (/Users/yuanyuanmiao/Documents/temp/egg-example/node_modules/cluster-client/lib/leader.js:77:23)
    at listOnTimeout (timers.js:324:15)
    at processTimers (timers.js:268:5)
name: "ClusterClientNoResponseError"
pid: 81333
hostname: yuanyuandeMacBook-Pro.local

